src/components/Header/HeaderContact.js

```diff
@@ -46,7 +46,7 @@ const HeaderContact = ({
   return (
     <View style={[styles.container, containerStyle]}>
       {renderHeader()}
-      <Text style={[styles.text, titleStyle]}>{title}</Text>
+      <Text style={[styles.text, styles.titleText, titleStyle]}>{title}</Text>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding `styles.titleText` to the Text component's style prop is a good move if you want to apply additional styles to the title text. However, please ensure that these new styles do not conflict with the existing `titleStyle` prop.
```diff
- <Text style={[styles.text, titleStyle]}>{title}</Text>
+ <Text style={[styles.text, styles.titleText, titleStyle]}>{title}</Text>
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->